docs/index.md

```diff
@@ -15,46 +15,50 @@ In order to integrate with Lead Safe, the health provider will need to make a RE
 ## API
 
 ### Submitting Prediction
-
+ 
 Submit a patient record and retrieved the estimated probability of having elevated blood-lead levels.
```

Comment:
"retrieved" should be "retrieve". I'd replace "probability" with "risk" since we haven't decided that it will return a true probability.

Reply:
Agreed.